test/e2e/attach_test.go

```diff
@@ -82,6 +82,7 @@ var _ = Describe("Podman attach", func() {
 		Expect(results.OutputToString()).To(ContainSubstring("test"))
 		Expect(podmanTest.NumberOfContainersRunning()).To(Equal(1))
 	})
+
```

Comment:
You could have cheated on the test checker.  :^)

Reply:
:-D this came from looking at test failures